GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -679,7 +671,14 @@ internal enum Command
             OpenFileWith = 14,
             EditFile = 15,
             AddSelectionToCommitMessage = 16,
-            CreateBranch = 17
+            CreateBranch = 17,
+            Refresh = 18,
+            SelectNext = 19,
+            SelectNext2 = 20,
+            SelectNext3 = 21,
+            SelectPrevious = 22,
+            SelectPrevious2 = 23,
+            SelectPrevious3 = 24,
```

Comment:
Can you please annotate what these are?

Reply:
Done in [abe3612](https://github.com/gitextensions/gitextensions/pull/9898/commits/abe36121aa186dd17b7755805b239cfb2afde9d0) if I got you right.

Reply:
My guess was that why there are SelectNext , SelectNext2, SelectNext3
There can only be one hotkey for each command and here there are three key combinations for the same thing.
Maybe add a comment about that regardless

Reply:
Or rather
```
   SelectNext = 19,
   SelectNext_AlternativeHotkey1 = 20,
   SelectNext_AlternativeHotkey2 = 21,
```
because these names are visible in the HotkeySettingsPage.